plugin.video.skyvideoitalia/resources/lib/addonutils.py

```diff
@@ -78,19 +79,15 @@ def setSetting(setting, value):
     ADDON.setSetting(id=setting, value=str(value))
 
 
-def showOkDialog(heading, line):
+def showOkDialog(line, heading=NAME):
     xbmcgui.Dialog().ok(heading, line)
 
 
 def createListItem(
         label='', params=None, label2=None,
-        thumb=None, fanart=None, poster=None, arts=None,
-        videoInfo=None, properties=None, isFolder=True,
+        thumb=None, fanart=None, poster=None, arts={},
+        videoInfo=None, properties={}, isFolder=True,
         path=None, subs=None):
-    if arts is None:
-        arts = {}
-    if properties is None:
-        properties = {}
```

Comment:
While on it maybe add offscreen=True  to bypass guilocks

Reply:
done.